Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -145,6 +145,7 @@ final class TouchBarWindow: NSPanel {
 			backing: .buffered,
 			defer: false
 		)
+		self.level = NSWindow.Level(rawValue: Int(CGWindowLevelForKey(.assistiveTechHighWindow)))
```

Comment:
This is good, but for absolute niceness, could you add this as a `static let` to `NSWindow.Level`? So we can just do `self.level = . assistiveTechHighWindow`. Maybe also add the other ones from `CGWindowLevelKey` that are missing in `NSWindow.Level`.